Sources/ArcGISToolkit/FloorFilter/SiteFacilityPromptViewController.swift

```diff
@@ -0,0 +1,272 @@
+//
+// Copyright 2021 Esri.
+
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+// http://www.apache.org/licenses/LICENSE-2.0
+
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import UIKit
+import ArcGIS
+import Foundation
+
+/// ViewController for the site and facility prompt
+internal class SiteFacilityPromptViewController: UIViewController {
```

Comment:
```suggestion
class SiteFacilityPromptViewController: UIViewController {
```
Also, please remove all `public` modifiers.